modules/hostap/src/supp_api.c

```diff
@@ -2408,6 +2408,7 @@ int supplicant_ap_wps_pbc(const struct device *dev)
 
```

Comment:
Could you change the commit subject to more appropriate `hostap: Fix beacon ...`. The `zephyr` prefix should only be used when submitting zephyr specific patches to hostap repository.